Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,23 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+## Exercise I
+
```

Comment:
Great breakdowns Julie, looks like you're building a strong understanding of time complexities 👍